andino_gazebo/launch/andino_with_ros_control.launch.py

```diff
@@ -0,0 +1,135 @@
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+"""Launch Gazebo with a world that has Andino."""
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import IncludeLaunchDescription, TimerAction, RegisterEventHandler
+from launch.conditions import IfCondition
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import Node
+from launch.actions import ExecuteProcess
+from launch.event_handlers import OnProcessExit
+from launch.substitutions import Command
+from launch_ros.parameter_descriptions import ParameterValue
+from xacro import process_file
+
+def get_robot_description(use_ros_control:str)->str:
+    doc = process_file(os.path.join(get_package_share_directory('andino_gazebo'), 'urdf', 'andino.gazebo.xacro'),
+                             mappings={'use_gazebo_ros_control': use_ros_control})
+    robot_desc = doc.toprettyxml(indent='  ')
+    folder=get_package_share_directory('andino_description')
+    robot_desc =robot_desc.replace('package://andino_description/','file://'+str(folder)+'/')
+    return robot_desc
+
+def generate_launch_description():
+    # Arguments
+    use_sim_time = LaunchConfiguration('use_sim_time')
+    use_sim_time_argument = DeclareLaunchArgument('use_sim_time',
+            default_value='true',
+            description='Use simulation (Gazebo) clock if true')
+    pkg_gazebo_ros = get_package_share_directory('gazebo_ros')
+    pkg_andino_gazebo = get_package_share_directory('andino_gazebo')
+    pkg_andino_control = get_package_share_directory('andino_control')
+    
+    # Include andino
+    include_andino = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            os.path.join(pkg_andino_gazebo, 'launch', 'spawn_robot.launch.py'),
+        ),
+        launch_arguments={'use_gazebo_ros_control':'true'}.items()
+    )
+
+    robot_description = get_robot_description('true')
+    controller_params_file = os.path.join(pkg_andino_control,'config','andino_controllers.yaml')
+
+    control_node = Node(
```

Comment:
There is no need to start a controller manager, since gazebo is already starting one. This is causing the process to die.